commands/helpers/utils/logger.js

```diff
@@ -0,0 +1,41 @@
+const log = require('npmlog');
+
+log.heading = 'jambo';
```

Comment:
I think it's ok to keep it as 'jambo' heading since it's technically a validation running from jambo? Changing the heading and style with the current setup from logger.js would override the functions (export with different log config) so it wouldn't maintain the jambo npm color and heading. Unless I use the npmlog functions directly `log.error` instead of the exported functions.